frontend/src/hooks/useGetEmailHistory/index.tsx

```diff
@@ -0,0 +1,21 @@
+import emailApis from '@api/domain/email';
+import QUERY_KEYS from '@hooks/queryKeys';
+import useClubId from '@hooks/service/useClubId';
+import { useQuery } from '@tanstack/react-query';
+
+interface UseGetEmailHistoryProps {
+  applicantId: number;
+}
+
+export default function useGetEmailHistory({ applicantId }: UseGetEmailHistoryProps) {
+  const clubId = useClubId().getClubId() || '';
+
+  const { data: { emailResponses: emailHistory } = { emailResponses: [] } } = useQuery({
+    queryKey: [QUERY_KEYS.EMAIL_HISTORY, clubId, applicantId],
+    queryFn: () => emailApis.history({ clubId, applicantId }),
+  });
```

Comment:
초기값은 initalData 속성을 사용하는게 어떨까요?
조금 더 직관적으로 느껴지는데, 어떻게 생각하실까용
```suggestion
  const {
    data: { emailResponses: emailHistory },
  } = useQuery({
    queryKey: [QUERY_KEYS.EMAIL_HISTORY, clubId, applicantId],
    initialData: { emailResponses: [] },
    queryFn: () => emailApis.history({ clubId, applicantId }),
  });
```

Reply:
추가로 이메일 보냈을때 해당 응답에 대한 revalidate가 추가되면 좋겠어요! 
혹은 refetch 요청 없이 캐싱값만 수정해도 되겠구용!

Reply:
응답에 대한 revalidate가 어떤 건지 이해하지 못했어요. 어떤 케이스가 있는지 설명해주실 수 있을까용?

Reply:
> 초기값은 initalData 속성을 사용하는게 어떨까요?

initialData 속성을 까먹고 있었네요. 반영하겠습니다~